server/views.py

```diff
@@ -89,3 +89,21 @@ def set_user_api_key(request):
         "data": api_key,
     }
     return Response(response, status=200)
+
+
+@api_view(["POST"])
+def set_s3_key(request):
+    key = request.data
+    s3_name = key["name"]
+    access_key = key["accessKey"]
+    secret_key = key["secretKey"]
+    set_frontier_s3_key(s3_name, access_key, secret_key)
+
+    response = {
+        "success": True,
```

Comment:
Don't need to send a success flag in the response, this is already communicated by the 200 status code